.gitignore

```diff
@@ -33,13 +33,15 @@ yarn-debug.log*
 *.log
 .envrc
 terraform/app_secrets.yml
+*token*
 
 #terraform files
 .terraform/
 tfplan
 *.tfplan
 *.plan
-*secrets.tfvars
+*secrets*
```

Comment:
Same here. We shouldn't ignore everything that has `secrets` in the file name.

Reply:
resolved...